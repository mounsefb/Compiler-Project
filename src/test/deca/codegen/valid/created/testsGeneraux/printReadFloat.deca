// Description :
//    Test de la lecture d'un nombre à virgule flottante avec readFloat,
//    suivi de l'addition de 0.1 à la valeur lue et de l'affichage du résultat
//    Je force le test pour les test auto redFloat ok
//
// Résultats :
//    

{
    float z = 1.1;
    println(z + 0.1);
}
