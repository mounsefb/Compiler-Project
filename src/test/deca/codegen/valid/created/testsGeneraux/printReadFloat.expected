1.20000e+00
